app/helpers/katello/hosts_and_hostgroups_helper.rb

```diff
@@ -123,6 +133,24 @@ def lifecycle_environment_options(host, options = {})
       all_options.html_safe
     end
 
+    def lifecycle_environment_options(host, options = {})
+      content_options(
+        host,
+        fetch_lifecycle_environment(host, options).try(:id),
+        :lifecycle_environment,
+        options
+      )
+    end
+
+    def content_source_options(host, options = {})
```

Comment:
It'd be nice to have a test which goes over content_options - maybe adding a simple test of content_source_options or lifecycle_environment_options would do the trick